bin/fle-dump

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# ==================================================================================================
+# Copyright 2014 Twitter, Inc.
+# --------------------------------------------------------------------------------------------------
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this work except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file, or at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==================================================================================================
+
+from collections import deque
+import os
+from threading import Thread
+import time
+import sys
+
+if os.getenv("ZKTRAFFIC_SOURCE") is not None:
+  sys.path.insert(0,  "..")
+  sys.path.insert(0, ".")
+
+from zktraffic.fle.sniffer import Sniffer
+
+import colors
+from twitter.common import app
+
+
+app.add_option('--iface', default='eth0', type=str)
+app.add_option('--port', default=3888, type=int)
+app.add_option('-c', '--colors', default=False, action='store_true')
+
+
+class Printer(Thread):
+  def __init__(self, colors):
+    super(Printer, self).__init__()
+    self.setDaemon(True)
+    self._queue = deque()
+    self._print = self._print_color if colors else self._print_default
+
+  def run(self):
+    while True:
+      try:
+        msg = self._queue.popleft()
+      except IndexError:
+        time.sleep(0.1)
+        continue
+
+      self._print(msg)
```

Comment:
Collapse this into:

try:
  self._print(self._queue.popleft())
except IndexError:
  time.sleep(0.1)

...?